packages/babel-plugin-polyfill-es-shims/src/mappings.ts

```diff
@@ -162,7 +158,7 @@ for (const name of [
   });
 }
 
-function createDescriptor(name, version, pkg = name.toLowerCase(), subfolder) {
+function createDescriptor(name, version, pkg = name.toLowerCase(), subfolder?) {
```

Comment:
manually had to make the last param optional in a few funcs below because it's not always passed in its usage. not sure why Flow hadn't errored on these before?